python/tests/api/writer/test_mlflow.py

```diff
@@ -0,0 +1,59 @@
+import os
+import shutil
+
+import mlflow
+import pytest
+
+import whylogs as why
+from whylogs.api.logger.result_set import ViewResultSet
+from whylogs.api.writer.mlflow import MlflowWriter
+
+
+class TestMlflowWriter(object):
+    @classmethod
+    def teardown_class(cls):
+        shutil.rmtree("mlruns")
+        shutil.rmtree("/tmp/artifact_downloads")
```

Comment:
why not also use ```tempfile``` in the tests?

Reply:
the tempfile on the implementation is writing on local temporarily, then logging on to mlflow. 
i'm not sure if you can not persist a file with mlflow's `log_artifact`

Reply:
`mlruns` is generated automatically by mlflow when creating a run and `/tmp/artifact_downloads` was created to download back a profile with the Client

Reply:
mlruns is standard for both systems, but maybe i can change the /tmp to be a directory created at the same level as mlruns. how does that sound?